tests/logs/delete/test_delete/test_delete_failed.txt

```diff
@@ -0,0 +1,7 @@
+[    INFO] Loading items from pub:https://fakepub.com?task-id=12345
+[    INFO] Deleting ami-rhcos1 in account aws-china-storage
+[    INFO] Failed to delete ami-rhcos1 in aws-china-storage:Random exception
+[    INFO] Deleting ami-aws1 in account aws-na
+[    INFO] Failed to delete ami-aws1 in aws-na:Random exception
+[    INFO] Collecting results
+[    INFO] Delete completed
```

Comment:
Another thing I didn't notice before... Is it possible to log the whole task as failed whenever some critical failure occurs?

```suggestion
[    INFO] Collecting results
[    INFO] Delete failed
```